test/plugins/features/movies/index.test.js

```diff
@@ -20,4 +21,86 @@ describe('movies integration', () => {
 
   });
 
+  describe('get', () => {
+
+    it('Gets all movies', () => {
```

Comment:
these are all good tests, but it looks like you're putting all the tests at the integration level and not the controller level. Most of these should go in controller.test.js, and then you can just one test per method (create/list) in this file to simulate the end to end API calls